src/java.xml/share/conf/jaxp-compat.properties

```diff
@@ -3,9 +3,27 @@
 #
 # ** Warning: unsafe configuration, use with security assessment **
 #
-# jaxp-compat.properties is a JAXP configuration file with compatible settings
-# prior to a strict configuration such as jaxp-strict.properties. In this
-# configuration, properties that have more restrictive settings as in the
+# This is one of the three configuration files provided in the JDK:
+#
+# jaxp.properties: this is the default configuration that the JDK uses to set
+# property values when XML factories are initiated.
+#
+# jaxp-strict.properties: this file resembles what will become the Secure-By-Default
+# configuration where a strict restriction is the default. This file allows
```

Comment:
strict restriction needs rewording.

Perhaps something that indicates that this property file provides settings that will be equivalent to that will be the default JAXP settings in a future release to make the use of JAXP more secure out of the box